frontend/packages/knative-plugin/src/utils/get-knative-icon.ts

```diff
@@ -11,19 +11,21 @@ import {
   EventSourceApiServerModel,
   EventSourceCamelModel,
   EventSourceKafkaModel,
+  EventSourcePingModel,
 } from '../models';
 
 export const getKnativeEventSourceIcon = (kind: string): string => {
-  switch (kind) {
-    case referenceForModel(EventSourceApiServerModel):
+  switch (kindForReference(kind)) {
+    case EventSourceApiServerModel.kind:
```

Comment:
Why did you change it to not use `referenceForModel`? A different version of the same event source can probably have a different icon.

Reply:
`referenceForModel` returns `sources.knative.dev~v1alpha1~PingSource` and i don't want check with to associated with version/apiGroup as we currently we are fetching these dynamically so rather should be based on Kind.